application/F3DOptionsParser.cxx

```diff
@@ -180,6 +181,24 @@ class ConfigurationOptions
   Dictionnary ConfigDic;
 };
 
+//----------------------------------------------------------------------------
+void ConfigurationOptions::LoadPlugins(const F3DAppOptions& appOptions) const
+{
+  try
+  {
+    f3d::engine::loadPlugin("core"); // always load core plugin
```

Comment:
I wonder if this is the right name, "core" plugin contains not "Core" the VTKExtensions module, but contains VTK provided readers, is that correct  ? Maybe rename it ? "default", "defaultReaders", "VTKReaders" ?

Reply:
native